library/common/config_template.cc

```diff
@@ -1,61 +1,14 @@
 /**
-  * Templated default configuration
-  *
-  * Note: Parts of this configuration are required for the direct API to function.
-  */
+ * Templated default configuration
+ *
+ * Note: Parts of this configuration are required for the direct API to function.
+ */
 const char* config_template = R"(
 static_resources:
-  listeners:
-  - address:
-      socket_address: {address: 0.0.0.0, port_value: 9000, protocol: TCP}
-    filter_chains:
-    - filters:
-      - name: envoy.http_connection_manager
-        config:
-          stat_prefix: base
-          route_config:
-            virtual_hosts:
-            - name: all
-              domains: ["*"]
-              routes:
-                - match: {prefix: "/"}
-                  route: {cluster: base}
-          http_filters:
-          - name: envoy.router
-  - address:
-      socket_address:
-        protocol: TCP
-        address: 0.0.0.0
-        port_value: 9001
-    filter_chains:
-    - filters:
-      - name: envoy.http_connection_manager
-        typed_config:
-          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
```

Comment:
Personally I think we should leave this here since we plan on supporting it. Maybe add an inline note instead?

Reply:
I think we should delete for now until we support it to minimize the amount of code rot we carry around. We can always come back in history and grab it.